brainframe/cli/translations/info.en.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  description: "Provides information on the current BrainFrame server
```

Comment:
I don't think `on` is the best preposition here. Potentially use `about`?

Reply:
Did it!